### PR TITLE
Pydeck: Update @jupyter-widgets/base to support JupyterLab v2

### LINKED ISSUE #4488: Deck.show() in pydeck 0.3.0 broken on JupyterLab >=2.0
#### Description

The function `pydeck.bindings.deck.Deck.show()` is broken in JupyterLab 2.0.0

#### Repro Steps

Install JupyterLab >=2.0.0 in Python, and install the current version of pydeck.

The following will produce the bug

```python
import pydeck as pdk

# 2014 locations of car accidents in the UK
UK_ACCIDENTS_DATA = ('https://raw.githubusercontent.com/uber-common/'
                     'deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv')

# Define a layer to display on a map
layer = pdk.Layer(
    'HexagonLayer',
    UK_ACCIDENTS_DATA,
    get_position=['lng', 'lat'],
    auto_highlight=True,
    elevation_scale=50,
    pickable=True,
    elevation_range=[0, 3000],
    extruded=True,                 
    coverage=1)

# Set the viewport location
view_state = pdk.ViewState(
    longitude=-1.415,
    latitude=52.2323,
    zoom=6,
    min_zoom=5,
    max_zoom=15,
    pitch=40.5,
    bearing=-27.36)

# Render
r = pdk.Deck(layers=[layer], initial_view_state=view_state)
r.show()
```

The error `model not found` will appear in the browser's developer console execution of `.show()`.

#### Environment (please complete the following information):
 - Framework Version: [e.g. deck.gl 6.3.0] deck.gl 8.1 + pydeck 0.3.0
 - Browser Version: [e.g. Chrome 71.0] Appears not to be browser-specific
 - OS: [e.g. Mac OS X 10.4] Mac OS X 10.15

#### Workarounds

Until we can fix the issue, one of the following may unblock you:

- Use `.to_html()` instead of `.show()`
- Downgrade JupyterLab
- Use a Jupyter Notebook instead

### PR DESCRIPTION
**Pull request by kylebarron**
Closes #4488

#### Change List
- As suggested [here](https://github.com/jupyter-widgets/ipywidgets/issues/2798#issuecomment-593716999), allows `@jupyter-widgets/base` to resolve to `3.x`.

Tested manually so far on JupyterLab 2.1.2 and Jupyter Notebook 6.0.3.

### CODE REVIEW

**Comment by ajduberstein** [comment]
Any idea if this breaks for earlier versions of JupyterLab?

**Comment by kylebarron** [comment]
Just tested in Jupyter Lab `1.2.14` and it worked fine.

File: modules/jupyter-widget/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@deck.gl/json": "8.2.0-alpha.1",
     "@deck.gl/layers": "8.2.0-alpha.1",
     "@deck.gl/mesh-layers": "8.2.0-alpha.1",
-    "@jupyter-widgets/base": "^2",
+    "@jupyter-widgets/base": "^2 || ^3",
```
Comment (Pessimistress) [inline modules/jupyter-widget/package.json:37]: This will never resolve to 2.x without a lock file. Either drop `^2` Or move it to `peerDependencies`.

Comment (kylebarron) [reply modules/jupyter-widget/package.json:37]: I thought it looked weird, but that's what was [suggested by a Jupyter developer](https://github.com/jupyter-widgets/ipywidgets/issues/2798#issuecomment-593716999).

I can remove `^2` and then test to make sure it still loads fine in Notebook and Lab 1.x and 2.x

**Comment by kylebarron** [comment]
I changed to `^3.0.0`, but now I'm getting 
```
Error displaying widget: model not found
```
errors in JupyterLab 1.2.14.

I'm guessing that `^2 || ^3` is used in the `jupyter labextension install` step, and `^2` is selected if JupyterLab 1.x and `^3` is selected if JupyterLab 2.x.

**Comment by kylebarron** [comment]
It does work on jupyter notebook, lab 1.x and lab 2.x with `^2 || ^3` in `peerDependencies`.

**Pull request merged**